### PR TITLE
Improve karmadactl command description

### LINKED ISSUE #2665: Improve karmadactl command description
At present, the description format of karmadactl subcommands is not uniform, some do not have Long descriptions, some descriptions are sentences and some are not. From the next version, we will generate readable documents for commands, so I think we need to complete and unify the description information format.

See the page：https://karmada.io/docs/next/reference/karmadactl/karmadactl-commands/karmadactl_index

### PR DESCRIPTION
**Pull request by lonelyCZ**
Signed-off-by: lonelyCZ <chengzhe@zju.edu.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Improve karmadactl command description

**Which issue(s) this PR fixes**:
Fixes #2665 

**Special notes for your reviewer**:
/cc @RainbowMango @Poor12 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/deinit/deinit.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ const (
 )
 
 var (
+	deInitLong = templates.LongDesc(`
+		Removes the Karmada control plane from the Kubernetes cluster.`)
```
Comment (Poor12) [inline pkg/karmadactl/deinit/deinit.go:26]: I found that some places use the singular form and some places use the plural form.
How about removing the `s` uniformly?

File: pkg/karmadactl/join/join.go [hunk 2]
```diff
@@ -21,8 +21,8 @@ import (
 )
 
 var (
-	joinShort = `Register a cluster to control plane`
-	joinLong  = `Join registers a cluster to control plane.`
+	joinLong = templates.LongDesc(`
+		Registers a cluster to Karmada control plane with PUSH mode.`)
```
Comment (Poor12) [inline pkg/karmadactl/join/join.go:25]: Push

File: pkg/karmadactl/deinit/deinit.go (hunk 1 above)
Comment (lonelyCZ) [reply pkg/karmadactl/deinit/deinit.go]: done

**Comment by RainbowMango** [comment]
@Poor12 
Do you have any other comments?

**Comment by Poor12** [comment]
> @Poor12 Do you have any other comments?

Generally LGTM

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**